# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelParams.ipdlh
# Commit: 2289e685031e
# Full Hash: 2289e685031e878fee593b86cea90025b43d1fe9
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P10 wait PHttpChannel::OnStartRequestSent for permission/cookie update from parent, r=mayhemer,necko-reviewers
#   
#   Depends on D77751
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77752
# ==============================================================================

diff -r 673030046319 -r 2289e685031e netwerk/protocol/http/HttpChannelParams.ipdlh
--- a/netwerk/protocol/http/HttpChannelParams.ipdlh	Tue Jun 30 00:33:54 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelParams.ipdlh	Tue Jun 30 00:34:06 2020 +0000
@@ -48,6 +48,7 @@
   nsCString appCacheGroupId;
   nsCString appCacheClientId;
   nsIReferrerInfo overrideReferrerInfo;
+  bool shouldWaitForOnStartRequestSent;
 };
 
 } // namespace ipc